CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 Swift 5.5
 ---------
 
+* Type names are no longer allowed as an argument to a subscript parameter that expects a metatype type
+
+```swift
```

Comment:
Sorry, @xedin, would you mind indenting these lines so that all of the text falls under the bullet point in the formatted document?

Reply:
Sure!

Reply:
https://github.com/apple/swift/pull/37451